website/pages/docs/job-specification/csi_plugin.mdx

```diff
@@ -66,6 +66,13 @@ job "plugin-efs" {
   # as a system job ensures all nodes in the DC have a copy.
   type = "system"
 
+  # only 1 plugin of a given type and ID should be deployed on
```

Comment:
should this also get a special note up above similar to the privilege one?

Reply:
Yeah, probably a good idea. I feel like CSI docs are getting to be more warnings than content, but that's the nature of their state at this point 😀